common/shlibs

```diff
@@ -4062,3 +4062,5 @@ libomalloc-0.9.6.so singular-4.2.1_1
 libsingular_resources-4.2.1.so singular-4.2.1_1
 libm4ri-0.0.20200125.so m4ri-20200125_1
 libm4rie-0.0.20200125.so m4rie-20200125_1
+libkdumpfile.so.8 libkdumpfile-0.4.0_1
+libLimeSuite.so.20.10-1 LimeSuite-20.10.0_1
```

Comment:
This probably isn't needed unless its used outside of `LimeSuite`

Reply:
it's needed for the SoapyLMS7 subpkg